examples/vnet/doca_gw_dpdk.c

```diff
@@ -1234,10 +1238,32 @@ void doca_gw_dpdk_destroy(uint16_t port_id)
 	struct doca_gw_pipeline *pipe;
 
 	port = doca_get_port_byid(port_id);
```

Comment:
test if port is not zero. this is user calling you

Reply:
done